packages/playground/assets/js/playground.js

```diff
@@ -242,4 +243,13 @@ function scrollSelectedMenuItemIntoView() {
       });
     }, 500);
   }
+}
+
+function createMetaTags() {
+  var metaTag = document.createElement("meta");
+  var controlName = document.querySelector(".control-header").innerText;
+  var controlTag = document.querySelector(".control-tag").innerText;
+  metaTag.name = "description";
+  metaTag.content = controlName.replace(/([A-Z])/g, " $1").trim() + ", " + controlTag.replace(/[<>]/g, "") + ": UI5  Web Components usage and API docs";
+  document.head.appendChild(metaTag);
```

Comment:
a build-time solution would be preferred, let's submit this as it is and discuss how to improve it separately.